src/components/BarChart/examples/4.grouped-with-legend.jsx

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
```

Comment:
I threw something into this PR that I'd been meaning to do for a while: removing the examples with embedded legends and replacing them with the examples showing how to use the `Legend` component directly.